Lesson_2_JSON_YAML_XML/future.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Starship",
+  "height-m": 120,
+  "weight-t": 1300, 
+  "value of development-bln$": "2-3",
```

Comment:
Not a mistake as it's still a valid JSON, although there're some general practices to avoid:

1) Spaces between parts of the key String -> `valueOfDevelopment` (aka camelCase) or `ValueOfDevelopment` (aka PascalCase, which is like camelCase but first letter is capitalized as well) or `value_of_development` (aka snake_case)
2) Special chars as a part of the key String as they can be weirdly parsed by some programming-language-dependent libraries -> `valueOfDevelopmentInBlnUSD" (as an example)
3) `"2-3"` is a valid String, although key says it's a "value of smth" so most of the people would expect `Interger` as a value -> so here we have two options: either make it a human-readable string like `"2-3 billion USD"` OR define a precise Integer value like `2000000000` so it can be parsed like a number, not a string